data-prepper-plugins/log-generator-source/src/test/java/com/amazon/dataprepper/plugins/source/loggenerator/LogGeneratorSourceConfigTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.source.loggenerator;
+
+
+
+import org.junit.jupiter.api.Test;
+
+import static com.amazon.dataprepper.plugins.source.loggenerator.LogGeneratorSourceConfig.DEFAULT_INTERVAL_SECONDS;
+import static com.amazon.dataprepper.plugins.source.loggenerator.LogGeneratorSourceConfig.DEFAULT_LOG_COUNT;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import java.time.Duration;
+
+class LogGeneratorSourceConfigTest {
+    @Test
+    void GIVEN_defaultLogGeneratorSourceConfig_WHEN_getIntervalCalled_THEN_defaultConstantReturned() {
+        assertThat(new LogGeneratorSourceConfig().getInterval(), equalTo(Duration.ofSeconds(DEFAULT_INTERVAL_SECONDS)));
+    }
+
+    @Test
```

Comment:
You can probably combine these tests into a single one titled `GIVEN_defaultLogGeneratorSource_THEN_returnExpectedValues` that tests all of the defaults.